common/changes/@microsoft/rush/enelson-fix-builtin-option_2022-01-05-17-09.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Pass builtin plugins to the internal Rush launcher",
```

```suggestion
      "comment": "",
```

This change isn't relevant to NPM package consumers.